apps/drupal/particle.theme

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/**
+ * @file
+ * Functions to support theming in the Particle theme.
+ */
+
+/**
+ * Implements hook_preprocess_region().
+ */
+function particle_preprocess_region__header(&$variables) {
+  // Blocks contains list of blocks we want to call individually in Twig.
+  // Example, {{ content }} => {{ branding }}
+  $blocks = [
+    'branding' => 'particle_sitebranding',
+  ];
+  foreach ($blocks as $key => $value) {
+    $pattern = '/^' . $value . '/';
+    foreach ($variables['elements'] as $el_key => $el_value) {
+      if(preg_match($pattern, $el_key)) {
```

Comment:
Let's talk about what's happening here. Regex at this stage is a smell that makes me uncomfortable.